index.html

```diff
@@ -0,0 +1,325 @@
+                </p>
+                <ul>
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javaScript</li>
+                </ul>
+                <div class="project">
+                    <button class="btn">See Project</button>
+                </div>
+            </div>
+            <div class="card-column">
+                <img src="./images/desktop_project_2.png" class="desktop_project_img"> 
+                <img src="./images/project2.png" class="mobile_project_img">      
+            </div>
+        </div>
+        
+        <div class="card">
+            <div class="card-column">
+                <img src="./images/desktop_project_3.png" class="desktop_project_img">
+                <img src="./images/project3.png" class="mobile_project_img">
+            </div>
+            <div class="card-column-text">
+                <h1>Facebook 360</h1>
+                <div class="position">
+                    <b>Facebook</b>
+                    <span class="dot">.</span>
+                    <span class="pos">Back End Dev</span>
+                    <span class="dot">.</span>
+                    <span class="pos">2015</span>
+                </div>
+                <p class="work-text">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul>
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javaScript</li>
+                </ul>
+                <div class="project">
+                    <button class="btn">See Project</button>
+                </div>
+            </div>
+           </div>
+        <div class="card">
+            <div class="card-column-text">
+                <h1>Uber Navigation</h1>
+                <div class="position">
+                    <b>Uber</b>
+                    <span class="dot">.</span>
+                    <span class="pos">Back End Dev</span>
+                    <span class="dot">.</span>
+                    <span class="pos">2015</span>
+                </div>
+                <p class="work-text">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul>
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javaScript</li>
+                </ul>
+                <div class="project">
+                    <button class="btn">See Project</button>
+                </div>
+            </div>
+            <div class="card-column">
+                <img src="./images/project4.png" class="mobile_project_img">
+                <img src="./images/desktop_project_4.png" class="desktop_project_img">       
+            </div>
+
+
+        
+          
+      
+
+
+          
+        </div>
+
+    </section>
+
+    <section class="about">
+        <div class="about-column">
```

Comment:
Kindly note that in the about myself section, the left block takes 40% and the right one the remaining 60% of the page's width, so kindly change that 🤗

Your design:

![image](https://user-images.githubusercontent.com/73714615/192993888-a0033023-bc64-41c8-bd23-ef3b9da8ee6a.png)

Figma design:

![image](https://user-images.githubusercontent.com/73714615/192994039-19c293a8-b94e-4041-9309-583cbd9b2f3d.png)